babel.config.js

```diff
@@ -1,5 +1,23 @@
 const presets = ['@babel/preset-env', '@babel/preset-react']
 
-module.exports = {
-  presets
+const testPresets = [
+  [
+    '@babel/preset-env',
+    {
+      targets: { node: 'current' }
+    }
+  ],
+  '@babel/preset-react'
+]
+
+module.exports = function(api) {
+  api.cache(true)
+
+  return {
+    env: {
+      development: { presets },
+      production: { presets },
+      test: { presets: testPresets }
```

Comment:
jest can't handle `async` on its own, so it needs to know to target the current version of node